### ISSUE TITLE
Support for Playing Livestream HLS EXT-X-VERSION:3

### ISSUE THREAD

**Issue by bcupac** [google/shaka-player#1808]
We are having a problem playing HLS livestreams: Our "manifest" looks like this:
```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-STREAM-INF:BANDWIDTH=230000,CODECS="avc1.4d001e,avc1.42000d,avc1.64000c,avc1.64001e,avc1.64001f,mp4a.40.2,mp4a.40.5"
0.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=440000,CODECS="avc1.4d001e,avc1.42000d,avc1.64000c,avc1.64001e,avc1.64001f,mp4a.40.2,mp4a.40.5"
1.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=700000,CODECS="avc1.4d001e,avc1.42000d,avc1.64000c,avc1.64001e,avc1.64001f,mp4a.40.2,mp4a.40.5"
2.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1200000,CODECS="avc1.4d001e,avc1.42000d,avc1.64000c,avc1.64001e,avc1.64001f,mp4a.40.2,mp4a.40.5"
3.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=2500000,CODECS="avc1.4d001e,avc1.42000d,avc1.64000c,avc1.64001e,avc1.64001f,mp4a.40.2,mp4a.40.5"
4.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=5000000,CODECS="avc1.4d001e,avc1.42000d,avc1.64000c,avc1.64001e,avc1.64001f,mp4a.40.2,mp4a.40.5"
5.m3u8
```

And we're getting an error:
```
main.js:695 Player error TypeError: Cannot read property 'split' of null    at Fg (hls_parser.js:1251)
    at mg (hls_parser.js:1318)
    at hls_parser.js:1015
```
Does Shaka support EXT-X-VERSION:3 ? 

Playing a version:
https://akamai-axtest.akamaized.net/routes/lapd-v1-acceptance/www_c4/Manifest.m3u8
works ok.

Testing using the shaka player demo https://shaka-player-demo.appspot.com/demo, with custom manifest option, yields the same error

**Comment by MilosRasic** [comment]
After some debugging, it seems that the manifest posted by @bcupac is parsed and handled correctly. The error occurs when parsing one of the manifests for a specific bandwidth.

`getRequiredTag()` for parses an EXTINF tag like this:

`#EXTINF:6.006,pid=181`

into this:

```
{
	id: 11,
	name: "EXTINF",
	attributes: [
		{
			name: "6.006,pid",
			value: "181"
		}
	],
	value: null
}
```

It's not parsing 6.006 as the value. Instead, it's considering it part of the attribute name.

**Comment by MilosRasic** [comment]
Created #1810 

Please review. The future of not serving another player in addition to Shaka to hundreds of thousands of users may depend on this :wink:

**Comment by joeyparrish** [comment]
Fixed by #1810.
